source/guides/non-denizen/resource-packs.md

```diff
@@ -0,0 +1,248 @@
+
+- blockstates - This is where each block-state of items are saved.
+- font - This is where font data is saved. This guide does not cover this.
+- models - This is where the model data and files are saved
+- textures - This is where the texture image files are saved
+- fonts - This is where your font data is saved
+- sounds - This is where your sounds are saved
+- optifine - This is where your optifine data is saved. This guide does not cover this.
+
+For Optifine support, it's recommended you join their discord and review their documentation at their [Github Source](https://github.com/sp614x/optifine/tree/master/OptiFineDoc/doc).
+
+#### Example File: `sounds.json`
+This file indexes where Minecraft should look for your sounds. Below is an example of a setup for two custom sounds, `defence_levelup0` and `defence_levelup1`.
+```json
+{
+  "entity.player.defence.level": {
+    "sounds": [
+      {
+        "name": "custom/defence_levelup0"
+      },
+      {
+        "name": "custom/defence_levelup1"
+      }
+    ],
+    "subtitle": "Excited Trumpet Noises"
+  }
+}
+```
+
+The first key is the name for the command; in this example, `entity.player.defence.level`
+The only data object within the command we need to specify are `sounds`. Optionally, you can specify the subtitle that displays if subtitles are enabled in-game. If you place multiple sounds within the `"sounds":[]` array, the sound will randomize between them based on their weight.
+
+For each file, you will need the data: `"name":"FILEPATH/FILENAME"`, excluding the file's extension.
+Optionally, you can manually adjust the following valid properties of the sound:
+1) volume - The volume the sound will be played as. 
+    - Default is `1.0`; Valid volume ranges from `0.0` to `1.0`; where `1.0` is the loudest it may be played at.
+    - The Volume value accepts higher values using Denizen's PlaySound, however not by increasing the volume. It increases the audible distance the sound may be heard from. Ie; volume 5 can be heard from five chunks away.
+2) pitch - The pitch the sound plays at, altered from it's original `.ogg` form.
+    - Default is `1.0`; Valid pitches range from `0.0` to `2.0`; where `0.0` is high-pitched and `1.0` will be low-pitched.
+3) weight - The chance that this sopund will be selected as opposed to randomly.
+    - Default is 0; Only accepts valid integers, not adjustable within Denizen.
+4) stream - Determines if the sound should be streamed from it's file.
+    - Default is false
+    - Recommended to set this value as `true` if the sound is longer than two seconds to avoid lag.
+    - Use this sparingly; it's not optimal to specify everything true.
+    - This is used with all music disks.
+5) preload - Determines if the sound should be loaded when loading the pack, as opposed to when the player plays the sound. 
+    - Default is false
+    - Used for ambient noises.
+6) attenuation_distance - Determines the reduction rate based on distance.
```

Comment:
should put \` on `attenuation_distance` and `type` and `stream` and all the others

Reply:
Also, for `attenuation_distance`, clarify the effect of higher vs lower values